Automattic/wp-calypso
# Subscribers: Redirect user if insufficient privileges

Related https://github.com/Automattic/dotcom-forge/issues/4708

## Proposed Changes

Redirects the user to My Home if they don't have sufficient privileges to visit Subscribers.

## Testing Instructions

1. Create a new Business site.
2. With the site owner, visit `/subscribers/<site>` and verify the page loads as expected.
3. Add a second user to the site as an author.
4. With the second user, visit `/subscribers/<site>` and verify you're redirected to My Home.
